test/SensorTest.php

```diff
@@ -4,6 +4,9 @@
  *
  * @package Test
```

Comment:
double package declarations should not happen. always enforce same order that introduces the least amount of changes (author, copyright, package, license)?

Reply:
...or drop package altogether as it's not used for anything (preferred).